docker-compose.yml

```diff
@@ -390,3 +393,19 @@ services:
       JAVA_OPTS: -Dfile.encoding=UTF-8
     volumes:
       - georchestra_datadir:/etc/georchestra
+      
+  rabbitmq:
+    image: docker.io/bitnami/rabbitmq:3.12
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f --user georchestra:georchestra localhost:15672/api/health/checks/local-alarms >/dev/null || exit 1"]
+      interval: 30s
+      timeout: 10s
+      retries: 10
+    environment:
+      - RABBITMQ_LOGS=-
+      - RABBITMQ_USERNAME=georchestra
+      - RABBITMQ_PASSWORD=georchestra
+    ports:
+      - '15672:15672'
```

Comment:
do we really need/want that ?

Reply:
Like discussed in https://github.com/georchestra/docker/pull/213#discussion_r1304220744, we do not want to expose rabbitmq ports to the outside world.